tests/utils_tests/test_autoreload.py

```diff
@@ -251,3 +254,15 @@ def test_resets_trans_real(self):
         self.assertEqual(trans_real._translations, {})
         self.assertIsNone(trans_real._default)
         self.assertIsInstance(trans_real._active, _thread._local)
+
+
+class TestRestartWithReloader(SimpleTestCase):
+    """"
+    With Python 2 on Windows, restart_with_reloader() coerces  environment
+    variables to str to avoid "TypeError: environment can only contain
+    strings" in Python's subprocess.py.
+    """
+    def test_environment(self):
+        os.environ['SPAM'] = 'spam'
```

Comment:
I know that this is only meant to land in `1.11.x` wonder if using `u''` prefixes/`force_text()` would make what we're testing more clear.